src/hooks/use-fetch-wpcom-sites/index.tsx

```diff
@@ -178,15 +190,14 @@ export const useFetchWpComSites = ( connectedSiteIds: number[] ) => {
 		return fetchSites();
 	}, [ fetchSites ] );
 
-	const syncSites = useMemo(
+	const syncSitesWithSyncSupportForSelectedSite = useMemo(
 		() => transformSiteResponse( rawSyncSites, memoizedConnectedSiteIds ),
 		[ rawSyncSites, memoizedConnectedSiteIds ]
 	);
 
 	return {
-		syncSites,
+		syncSites: syncSitesWithSyncSupportForSelectedSite,
 		isFetching: isFetchingSites.current,
-		isInitialized: isInitialized.current,
 		refetchSites,
```

Comment:
```suggestion
		refetchSites: fetchSites,
```
Nit, but I think we can assign `fetchSites` directly to this key.